manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/AbstractWorker.java

```diff
@@ -30,6 +33,9 @@
 
 public abstract class AbstractWorker<T extends ManagedResource> implements Worker<T> {
 
+    private static final String RETRIES_FAILURE_MESSAGE = "The maximum number of re-tries for Resource of type '%s' with Id '%s' was exceeded.";
+    private static final String TIMEOUT_FAILURE_MESSAGE = "The timeout to process Work for Resource of type '%s' with Id '%s' was exceeded.";
+
```

Comment:
What about moving these constants to the classes `ProvisioningMaxRetriesExceededException` and `ProvisioningTimeOutException`?

Reply:
Done.